src/planscape/forsys/get_forsys_inputs.py

```diff
@@ -20,26 +23,37 @@ class ForsysInputHeaders():
     # Only used for generation, not prioritization.
     FORSYS_GEO_WKT_HEADER = "geo"
 
+    # Note: when each stand represents a pixel, conditions and priorities will
+    # have the same values; however, when a single stand represents multiple
+    # pixels, conditions and priorities will have diffent values.
+    # Conditions represent the mean score across all pixels within a stand.
+    # This is useful when coloring maps.
+    # Priorities represent the sum of scores across all pixels within a stand.
+    # This is useful when scoring stands and project areas in Forsys.
+
     # Header prefixes for conditions and priorities.
-    _CONDITION_PREFIX = "c_"
+    _CONDITION_PREFIX = "c_"  # Only used for generation, not prioritization.
```

Comment:
why is it only used for generation?

Reply:
The header is used in both generation and prioritization.

The condition prefix is only alluded to in the generation logic.

At the moment, condition scores are more of a debug tool when coloring maps, and there isn't much need to debug prioritization because there's nothing to fine-tune for priotization.